pkg/integration/tests/stash/drop_multiple.go

```diff
@@ -0,0 +1,49 @@
+package stash
+
+import (
+	"github.com/jesseduffield/lazygit/pkg/config"
+	. "github.com/jesseduffield/lazygit/pkg/integration/components"
+)
+
+var DropMultiple = NewIntegrationTest(NewIntegrationTestArgs{
+	Description:  "Drop multiple stash entries",
+	ExtraCmdArgs: []string{},
+	Skip:         false,
+	SetupConfig:  func(config *config.AppConfig) {},
+	SetupRepo: func(shell *Shell) {
+		shell.EmptyCommit("initial commit")
+		shell.CreateFile("file1", "content1")
+		shell.GitAddAll()
+		shell.Stash("stash one")
+		shell.CreateFile("file2", "content2")
+		shell.GitAddAll()
+		shell.Stash("stash two")
+		shell.CreateFile("file3", "content3")
+		shell.GitAddAll()
+		shell.Stash("stash three")
+	},
+	Run: func(t *TestDriver, keys config.KeybindingConfig) {
+		t.Views().Files().IsEmpty()
+
+		t.Views().Stash().
+			Focus().
+			Lines(
+				Contains("stash three").IsSelected(),
```

Comment:
I'd prefer a test that doesn't start from the first stash; just create one more and delete from the second on down (and maybe just the middle two of four).